service/geopm-service.spec.in

```diff
@@ -107,6 +107,12 @@ Group: System Environment/Libraries
 Group: System/Libraries
 %endif
 
+# Optional dependency: Remove --enable-io-uring from the ./configure step. 
+# Lets GEOPM batch its IO operations to reduce syscall overhead in IOGroups
+# with a lot of reads/writes per GEOPM batch operation.
+BuildRequires: liburing-devel
+Requires: liburing
+
```

Comment:
```suggestion
# Lets GEOPM batch its IO operations to reduce syscall overhead in IOGroups
# with a lot of reads/writes per GEOPM batch operation.
BuildRequires: liburing-devel

```

Reply:
The requires will be automatically be picked up by rpmbuild when the elf header is inspected.